src/test/ui/nll/lint-no-err.rs

```diff
@@ -0,0 +1,23 @@
+// check-pass
```

Comment:
oh is it because of `check-pass`

Reply:
but i don't see why it wouldn't be picked by the CI then

Reply:
this borrowcheck warning is a hard error with `nll` enabled, we don't test nll in ci